### DISCUSSION TITLE
Using ISAR DB as database server to access from other mobile devices in Local Area Network

### DISCUSSION THREAD

**Discussion by nihatdemir79** [isar/isar#1619] [Help]
Hi guy,

I have project and I need to have database server (Server App) in Android to server data to other apps (Client apps) to write/read data by using TCP/IP in Local Network. If client apps cant reach the Server App, they will stop working. Can I do user ISAR for this project. If not what would you suggest?

Many Thanks

Nihat

**Comment by azkadev** [comment]
yes you can, but you need to install the native isar library manually
